linera-service/src/client.rs

```diff
@@ -51,6 +51,13 @@ pub enum Network {
     Simple,
 }
 
+#[derive(Copy, Clone)]
+pub enum Database {
+    RocksDb,
```

Comment:
There is also memory I suppose.

Reply:
Some tests are passing with `Memory` but not all.
I think adding `Memory would lead us to too many issues that distract from the objective of the end to end tests.

Reply:
The difference between RocksDb and Memory should only be visible in tests that restarts validator shards.

Reply:
Ok, done.